content/en/docs/concepts/semantic-conventions.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Semantic Conventions
+description: >-
+  Common names for different kinds of operations and data
+weight: 30
+---
+
+OpenTelemetry defines Semantic Conventions (sometimes called Semantic Attributes) that specify common names for different kinds of operations and data. The benefit to using Semantic Conventions is in following a common naming scheme that can be standardized across a codebase, libraries, and platforms.
+
+Currently, Semantic Conventions are split by signal:
```

Comment:
ppl might get confused that there is a "Resource Semantic Convention" then, maybe

```suggestion
Currently, Semantic Conventions are available for traces, metrics and resources:
```